runtime/restart/monitor/monitor.go

```diff
@@ -99,6 +100,9 @@ func (m *monitor) run(interval time.Duration) {
 }
 
 func (m *monitor) reconcile(ctx context.Context) error {
+	if m.reconcileTimes >= 0 && m.reconcileTimes < 2 {
+		m.reconcileTimes++
+	}
```

Comment:
Hard to read this

Reply:
I write old version like 
```
func (m *monitor) reconcile(ctx context.Context) error {
m.reconcileTimes++
//then check it out of limit
if m.reconcileTimes == math.MaxInt64 {
m.reconcileTimes = 2
}
...
}
```
but I found  m.reconcileTimes doesn't to plus 1 at every time, I just need  m.reconcileTimes=1 ,so I use 
```
	if m.reconcileTimes >= 0 && m.reconcileTimes < 2 {
		m.reconcileTimes++
	}
```
instead